packaging/installer/functions.sh

```diff
@@ -765,15 +763,16 @@ NETDATA_ADDED_TO_GROUPS=""
 add_netdata_user_and_group() {
 	local homedir="${1}" g
```

Comment:
I am a bit confused, what are we declaring here? what is g?

Reply:
no idea and this is not a part of PR.